docs/users.rst

```diff
@@ -1248,8 +1248,10 @@ available in the OSBS buildroot as `bsi` executable in `$PATH`.
 
 
 Current limitations:
-* only Source RPMs are added into source container image
+
+* only Source RPMs and sources fetched through cachito integration are added into source container image
```

Comment:
Is possible to link `cachito` to cachito section?

Reply:
added